tools/browse_dataset.py

```diff
@@ -127,16 +127,17 @@ def retrieve_data_cfg(config_path, skip_type, show_origin=False):
     train_data_cfg = cfg.data.train
```

Comment:
Can also add an argument named `options`, refer to https://github.com/open-mmlab/mmdetection/blob/f08548bfd6/tools/misc/browse_dataset.py#L36 and https://github.com/open-mmlab/mmsegmentation/blob/master/tools/train.py#L71.